<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graficas Matplotlib</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border-radius:20px;
border: 2px solid #000000;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,5">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">QLabel{

border: none;
}</string>
            </property>
            <property name="text">
             <string>Grafica N째 1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="grafica_uno"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border-radius:20px;
border: 2px solid #000000;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">QLabel{

border: none;
}</string>
            </property>
            <property name="text">
             <string>Grafica N째 2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="grafica_dos"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_4">
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border-radius:20px;
border: 2px solid #000000;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">QLabel{

border: none;
}</string>
            </property>
            <property name="text">
             <string>Grafica N째 3</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="grafica_tres"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="frame_5">
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border-radius:20px;
border: 2px solid #000000;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">QLabel{

border: none;
}</string>
            </property>
            <property name="text">
             <string>Grafica N째 4</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="grafica_cuatro"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
